.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,22 @@
+name: Vulnerabilities scanning of dependencies using Snyk
+on:
+  push:
+    branches: main
```

Comment:
I følge dokumentasjonen skal det være linjeskift og bindestrek før banchnavnet, se https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters